googleMapsApiModule.py

```diff
@@ -2,15 +2,17 @@
 import webbrowser
 import config
 import mysql.connector
+from constants import *
```

Comment:
It's usually preferred to import just specific values (like, `BASE_URL` here) rather than `*` - because:

1. it'll ensure you don't import any unnecessary value
2. it'll allow you to backtrack through where the variable came from (in your IDE)

Reply:
yeah I thought of it but as it's having only one dictionary, so I imported all.

Reply:
I will change it.